src/components/ngRedux.js

```diff
@@ -78,7 +78,8 @@ export default function ngReduxProvider() {
       ? applyMiddleware(...resolvedMiddleware)(finalCreateStore)(_reducer, _initialState)
       : applyMiddleware(...resolvedMiddleware)(finalCreateStore)(_reducer);
 
-    return assign({}, store, { connect: Connector(store) });
+    store.connect = Connector(store);
```

Comment:
It looks like `_.assign` does not copy over Symbols. Can you change this PR to remove `assign` and use `Object.assign` instead? That *should* copy it over while keeping everything immutable as before.